### ISSUE TITLE
Apply NMDC color scheme (via Ionic theme)

### ISSUE THREAD

**Issue by eecavanna** [microbiomedata/nmdc-field-notes#14]
Here's a screenshot of part of the NMDC Branding Notes document:

<img width="810" alt="image" src="https://github.com/microbiomedata/nmdc-field-notes/assets/134325062/7d18aed3-1b8c-42ed-ac6e-a54abbf8d2d8">

Here are the hex color codes shown in that screenshot:
- Blue: `#00AAE7`
- Purple: `#4F3B80`
- Red: `#ED5338`
- Orange: `#E88320`

Here's a link to the Ionic color generator: https://ionicframework.com/docs/theming/color-generator

Other references:
- Color definitions in `nmdc-server`: https://github.com/microbiomedata/nmdc-server/blob/main/web/src/colors.js
    - Red, Orange, Blue, and Purple are the same as above (i.e. the NMDC Branding Guidelines).
    - A Green is also defined. I think it has the value of the `green-darken-2` color at https://vuetifyjs.com/en/styles/colors/#material-colors
        - That is `#388E3C`
- Color definitions in `nmdc-status-website`: 
    - https://github.com/microbiomedata/nmdc-status-website?tab=readme-ov-file#color-palette 
    - I inferred these from the NMDC website

**Comment by eecavanna** [comment]
The [Ionic color generator](https://ionicframework.com/docs/theming/color-generator) allows me to specify the following colors:
- Primary
- Secondary
- Tertiary
- Success
- Warning
- Danger
- Light
- Medium
- Dark

I don't know whether the NMDC Data Portal developers have already associated those color "roles" with the specific colors in the NMDC Branding Guidelines. For example, maybe they consider the "Blue" color to be the "Primary" color.

**Comment by eecavanna** [comment]
Here's what _one_ way of allocating those colors looks like:

![image](https://github.com/microbiomedata/nmdc-field-notes/assets/134325062/3582ca5c-76f7-4df2-8dd9-335e354b305c)

Note that the "Warning" and "Danger" labels happen to be below the red and orange rectangles, respectively. I think their visual proximity in the preview is just a coincidence. Each of the four things gets its color from a different variable.

**Comment by eecavanna** [comment]
A preliminary solution for this has been implemented in commit e530cf8c63c7d0058a1a3e67b5b21de50504dfa0.

**Comment by simroux** [comment]
Looking good ! The only thing I had was : according to the [Social Media Playbook](https://docs.google.com/document/d/1sW-gyv5RbnHiSiBQNb8U8b4qwLi_XP1CKV_IRngZ9zc/edit) "NMDC Submission Portal is illustrated with blue hues". Since the Field Notes App is primarily linked to the submission portal, maybe we can switch primary vs secondary so that the App is predominantly blue ?

**Comment by eecavanna** [comment]
Thanks for bringing that statement to my attention. I hadn't noticed it before.

I looked at the submission portal just now and it seems to me to use the colors in the same way as the data portal, where the primary color is purple.

<img width="997" alt="image" src="https://github.com/microbiomedata/nmdc-field-notes/assets/134325062/e1df20b0-4c56-4cc8-8454-5168882c1b21">

I'll create a new ticket about rearranging the color palette (i.e. swapping designations/color roles) from what I showed above, which I'll call the "preliminary color palette." I'll link it to this ticket so readers of the new ticket can find this additional context.
